### DISCUSSION TITLE
Installing river in a kaggle.com notebook

### DISCUSSION THREAD

**Discussion by devcon14** [online-ml/river#809] [Q&A]
I'm keen to use river in a kaggle contest.

After pip install of river 0.9.0 I get this error when importing river in the environment:
python==3.7.0
numpy==1.19.5

RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd
ImportError: numpy.core.multiarray failed to import

Is there likely to be an upgrade soon or does anyone know of a workaround to link to an older numpy?

**Comment by MaxHalford** [comment]
Hey there.

I believe you need to run this:

```sh
pip install numpy --upgrade
```

after you've installed River. Then you can import River. If it doesn't work, then click on `Run > Restart & clear cell outputs` and try to import River once again. 

Hope that helps. I tried it and it worked for me.

**Reply by devcon14** [reply]
That seems to do the trick.

Not sure if needing to run ```Restart & clear cell outputs``` after installation might effect code submissions but I'll do some experimentation and see how it goes.

Thanks.
